### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Closes https://github.com/dbt-labs/docs-internal/issues/754
Closes https://github.com/dbt-labs/docs-internal/issues/753
Closes https://github.com/dbt-labs/docs-internal/issues/752

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-release-notes-dbt-labs.vercel.app/docs/dbt-versions/release-notes

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes.md [hunk 1]
```diff
@@ -19,7 +19,10 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 \* The official release date for this new format of release notes is May 15th, 2024. Historical release notes for prior dates may not reflect all available features released earlier this year or their tenancy availability.
 
 ## October 2024
-
+ 
+- **Fix:** Previously, POST requests to the Jobs API with invalid cron strings would return 500s but would update the underlying entity. Now, POST requests to the Jobs API with invalid cron strings will result in 400s, without the underlying entity being updated.
+- **Fix:** Corrected an issue where the `Source` view page in Explorer did not properly display source frehsness status if older than 30 days.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes.md]:
```suggestion
- **Fix:** Fixed an issue where the `Source` view page in dbt Explorer did not correctly display source freshness status if older than 30 days.
```

File: website/docs/docs/dbt-versions/release-notes.md [hunk 2]
```diff
@@ -19,7 +19,10 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
...
+- **Fix:** The UI now indicates when the description of a model is inhereted from a catalog comment.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes.md]:
```suggestion
- **Fix:** The UI now indicates when the description of a model is inherited from a catalog comment.
```

File: website/docs/docs/dbt-versions/release-notes.md [hunk 3]
```diff
@@ -19,7 +19,10 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 \* The official release date for this new format of release notes is May 15th, 2024. Historical release notes for prior dates may not reflect all available features released earlier this year or their tenancy availability.
 
 ## October 2024
-
+ 
+- **Fix:** Previously, POST requests to the Jobs API with invalid cron strings would return 500s but would update the underlying entity. Now, POST requests to the Jobs API with invalid cron strings will result in 400s, without the underlying entity being updated.
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes.md]:
```suggestion
- **Fix:** Previously, POST requests to the Jobs API with invalid `cron` strings would return HTTP response status code 500s but would update the underlying entity. Now, POST requests to the Jobs API with invalid `cron` strings will result in status code 400s, without the underlying entity being updated.
```

**Review by runleonarun — APPROVED** [review]
One tiny suggestion!

**Pull request merged**